lib/internal/async_hooks.js

```diff
@@ -345,6 +365,18 @@ function emitDestroyScript(asyncId) {
 }
 
 
+function clearAsyncIdStack() {
```

Comment:
nit: needs a comment that needs to stay in sync with the C++ copy in `env-inl.h` (and vice versa).

Reply:
Good point. Will update!